torch/distributions/bernoulli.py

```diff
@@ -36,6 +36,7 @@ class Bernoulli(ExponentialFamily):
     Args:
         probs (Number, Tensor): the probability of sampling `1`
         logits (Number, Tensor): the log-odds of sampling `1`
+        validate_args (bool): whether to validate arguments, None by default
```

Comment:
`(bool)` -> `(bool, optional)` 
for here and below?

Reply:
Thanks for your review! Have updated in this commit: https://github.com/pytorch/pytorch/pull/152173/commits/6e8e0de1beeea9dc41f8ae9e48246f198a866054